tests/test_sdist.py

```diff
@@ -50,10 +50,12 @@ def test_ignore_based_on_marker(tmp_context: WorkContext):
     assert version == ""
 
 
-def test_invalid_wheel_file_exception(tmp_path: pathlib.Path):
+@patch("fromager.sources.download_url")
+def test_invalid_wheel_file_exception(mock_download_url, tmp_path: pathlib.Path):
+    mock_download_url.return_value = str(tmp_path / "test" / "fake_wheel.txt")
```

Comment:
I think this function is expected to return a `pathlib.Path()`, isn't it?

Reply:
Yeah missed it, fixing now